package.json

```diff
@@ -27,6 +28,7 @@
     "test:core:ci": "nyc --include \"packages/dd-trace/src/**/*.js\" -- npm run test:core",
     "test:plugins": "mocha --exit --file \"packages/dd-trace/test/setup/core.js\" \"packages/datadog-plugin-@($(echo $PLUGINS))/test/**/*.spec.js\"",
     "test:plugins:ci": "yarn services && nyc --include \"packages/datadog-plugin-@($(echo $PLUGINS))/src/**/*.js\" -- npm run test:plugins",
+    "test:plugins:debug": "yarn test:plugins --inspect-brk",
```

Comment:
it came in handy when debugging plugin tests, because I could use the dev tools in chrome. In any case, this can be done easily via `yarn test:plugins --inspect-brk` so I don't think it is necessary. I haven't tried `yarn tdd`. I'll give it a look. 
Removing this 👍